packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -2,74 +2,69 @@ import { useMedia } from 'react-use'
 import dayjs, { Dayjs } from 'dayjs'
 
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
-import { ChainId } from '../../util/networks'
+import {
+  getBaseChainIdByChainId,
+  getBlockTime,
+  getConfirmPeriodBlocks
+} from '../../util/networks'
 import { MergedTransaction } from '../../state/app/state'
 
 /**
- * Buffer for after a node is confirmable but isn't yet confirmed; we give 30 minutes, should usually/always be less in practice.
+ * Buffer for after a node is confirmable but isn't yet confirmed.
+ * A rollup block (RBlock) typically gets asserted every 30-60 minutes.
  */
-const CONFIRMATION_BUFFER_MINUTES = 30
+const CONFIRMATION_BUFFER_MINUTES = 60
 
-function getTxConfirmationDate({
-  createdAt,
-  parentChainId
-}: {
-  createdAt: Dayjs
-  parentChainId: number
-}) {
-  const confirmNodeMinutes = getNodeConfirmationTimeInMinutes(parentChainId)
-
-  return createdAt
-    .add(confirmNodeMinutes, 'minute')
-    .add(CONFIRMATION_BUFFER_MINUTES, 'minute')
-}
+const SECONDS_IN_MIN = 60
 
-function getTxConfirmationRemainingMinutes({
+export function getTxConfirmationDate({
   createdAt,
-  parentChainId
+  withdrawalFromChainId,
+  baseChainId
 }: {
-  createdAt: Dayjs
-  parentChainId: number
+  createdAt?: Dayjs | null
```

Comment:
Doesn't look like we ever skip this value when passing props, or pass undefined/null. Can we just use `createdAt: Dayjs`?